pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -19,12 +19,11 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog"
 	"k8s.io/kubectl/pkg/drain"
-	kubevirtv1alpha1 "kubevirt.io/node-maintenance-operator/pkg/apis/kubevirt/v1alpha1"
+	kubevirtv1beta1 "kubevirt.io/node-maintenance-operator/pkg/apis/kubevirt/v1beta1"
```

Comment:
suggest we simply import as `kubevirt` or more appropriately as `maintenance` (or similar)

Reply:
changed to nodemaintenanceapi

Reply:
done